<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="mainText" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>650</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QLabel" name="tileLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>27</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="titleText">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>9</y>
       <width>555</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
    <widget class="QLabel" name="authorLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>95</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Authors</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="authorText">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>95</y>
       <width>555</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="journelLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>182</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Conference</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="journelText">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>182</y>
       <width>555</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>268</y>
       <width>28</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="dateText">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>268</y>
       <width>555</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="tagLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>355</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tag</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tagText">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>355</y>
       <width>555</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="abstractLable">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>441</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Abstract</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="abstractText">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>441</y>
       <width>555</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="tabStopWidth">
      <number>100</number>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>331</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="port">
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>23</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>mainText</zorder>
   <zorder>listWidget</zorder>
   <zorder>findButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
